packages/ecommerce-ui/components/FormComponents/CheckboxElement.tsx

```diff
@@ -12,12 +12,16 @@ import {
   FormControlLabelProps,
   FormGroup,
   FormHelperText,
+  SxProps,
+  Theme,
 } from '@mui/material'
 
 export type CheckboxElementProps<T extends FieldValues> = Omit<CheckboxProps, 'name'> & {
   parseError?: (error: FieldError) => string
   label?: FormControlLabelProps['label']
   helperText?: string
+  sx?: SxProps<Theme>
+  sxFormControl?: SxProps<Theme>
```

Comment:
Might it be an idea to do something like:

```
formControl?: Omit<FormControlProps<'div'>, 'required' | 'error'>
```

Reply:
Good idea! Done :)